aiida/cmdline/commands/computer.py

```diff
@@ -13,109 +13,151 @@
         try:
-            getter = dict(inspect.getmembers(computer))[getter_name]
-        except KeyError:
-            print >> sys.stderr, ("Internal error! " "No {} getter defined in Computer".format(getter_name))
-            sys.exit(1)
-        previous_value = getter()
+            SchedulerClass = scheduler_ep.load()
+        except ImportError:
+            raise ImportError("Unable to load the '{}' scheduler".format(scheduler_ep.name))
+    else:
+        raise ValidationError(
+            "The shouldcall_... function should always be run (and prompted) AFTER asking for a scheduler")
 
-        setter_name = '_set_{}_string'.format(internal_name)
-        try:
-            setter = dict(inspect.getmembers(computer))[setter_name]
-        except KeyError:
-            print >> sys.stderr, ("Internal error! " "No {} setter defined in Computer".format(setter_name))
-            sys.exit(1)
+    JobResourceClass = SchedulerClass._job_resource_class
+    if JobResourceClass is None:
+        # Odd situation...
+        return False
 
-        valid_input = False
-        while not valid_input:
-            if multiline:
-                newlines = []
-                print "=> {}: ".format(name)
-                print "   # This is a multiline input, press CTRL+D on a"
-                print "   # empty line when you finish"
+    return JobResourceClass.accepts_default_mpiprocs_per_machine()
 
-                try:
-                    for l in previous_value.splitlines():
-                        while True:
-                            readline.set_startup_hook(lambda: readline.insert_text(l))
-                            input_txt = raw_input()
-                            if input_txt.strip() == '?':
-                                print ["  > {}".format(descl) for descl in "HELP: {}".format(desc).split('\n')]
-                                continue
-                            else:
-                                newlines.append(input_txt)
-                                break
-
-                    # Reset the hook (no default text printed)
-                    readline.set_startup_hook()
-
-                    print "   # ------------------------------------------"
-                    print "   # End of old input. You can keep adding     "
-                    print "   # lines, or press CTRL+D to store this value"
-                    print "   # ------------------------------------------"
-
-                    while True:
-                        input_txt = raw_input()
-                        if input_txt.strip() == '?':
-                            print "\n".join(["  > {}".format(descl) for descl in "HELP: {}".format(desc).split('\n')])
-                            continue
-                        else:
-                            newlines.append(input_txt)
-                except EOFError:
-                    # Ctrl+D pressed: end of input.
-                    pass
+## Todo: reenable interactive and mixed tests
```

Comment:
They look enabled to me, please double-check and remove this line

Reply:
ok